Formula/o/openfortivpn.rb

```diff
@@ -20,10 +20,13 @@ class Openfortivpn < Formula
   depends_on "pkg-config" => :build
   depends_on "openssl@3"
 
+  uses_from_macos "pppd"
```

Comment:
There is no formula for that yet, so maybe just comment it out?
```suggestion
  # awaiting formula creation
  # uses_from_macos "pppd"
```

Reply:
Sure, I can do that. Time permitting, I could try to create a “system” formula for ppp/pppd, based on other system formulae such as [Formula/z/zlib.rb](https://github.com/Homebrew/homebrew-core/blob/master/Formula/z/zlib.rb), but that's probably beyond what I can do without owning a mac.